torch/_torch_docs.py

```diff
@@ -8174,6 +8174,42 @@ def merge_dicts(*dicts):
 """,
 )
 
+add_docstr(
+    torch.narrow_copy,
+    r"""
```

Comment:
The online documentation has an entry for `narrow_copy` https://pytorch.org/docs/stable/generated/torch.Tensor.narrow_copy.html

I'm not sure what source it is being generated from as there was no docstring before adding this. We probably want to see if @Lezcano knows where that documentation is coming from and how that would interact with addition of a docstring here. 

Also it is probably a good idea to merge the information somehow, specifically mentioning the motivation for the method is sparse tensor support and how the function interacts with the sparse/dense dimension partition for sparse tensors.

Reply:
Note that the current documentation is docs for a `Tensor` method. In any case, we can grep a bit of the text as `git grep "except returning a copy rathe"` to find that it lives in `torch/_tensor_docs.py`.

When we have a function that is both a method and a function (as it happens with 99% of operators in PyTorch) we often put the docs in the function, and write a short pointer to the function docs in the method. See for example https://pytorch.org/docs/stable/generated/torch.Tensor.deg2rad.html

Reply:
updated